src/platforms/python/common/performance/instrumentation/custom-instrumentation.mdx

```diff
@@ -106,6 +106,35 @@ def eat_slice(slice):
     swallow()
 ```
 
+## Define Span Creation in a Central Place
+
+If you do not want to have custom performance instrumentation code scattered all over your code base, you can pass a parameter <PlatformIdentifier name="functions-to-trace" /> to your `sentry_sdk.init()` call.
```

Comment:
```suggestion
To avoid having custom performance instrumentation code scattered all over your code base, pass a parameter <PlatformIdentifier name="functions-to-trace" /> to your `sentry_sdk.init()` call.
```